plugins/catalog/src/components/actions.ts

```diff
@@ -75,3 +75,25 @@ export const determineUrlType = (url: string): string => {
   }
   return 'url';
 };
+
+export const findEditUrl = (
+  { metadata }: Entity,
+  location?: LocationSpec,
+): string | undefined => {
+  const annotations = metadata.annotations || {};
+
+  const editUrl = annotations['backstage.io/browser-edit-url'];
+
+  if (editUrl) return editUrl;
+
+  return location && createEditLink(location);
+};
+
+export const findViewUrl = (
+  { metadata }: Entity,
+  location?: LocationSpec,
+): string | undefined => {
+  const annotations = metadata.annotations || {};
+
+  return annotations['backstage.io/browser-view-url'] || location?.target;
```

Comment:
So i'm thinking ... this logic (and same above),

Just move the `findLocationForEntityMeta` call in here, and remove the location argument. And since these are so browser specific, maybe we should only return something when that returns a `url` type and the target starts with `http:` or `https:`? Otherwise we can just return undefined.

Reply:
The only caveat would be that you're going to do the `findLocationForEntityMeta` twice on the AboutCard I think. But it shouldn't make any real difference.

Reply:
No problem. And could be refactored to calculate both in one go if desired :)